code/datums/brain_damage/psychological.dm

```diff
@@ -0,0 +1,330 @@
+	..()
+
+/datum/brain_trauma/psychological/bipolar/proc/change_mood_swing()
+	mood_swing_state = !mood_swing_state
+	if(mood_swing_state)
+		to_chat(owner, "<span class='danger'>Your heart begins to race. You are overfilled with joy!.</span>")
+		SEND_SIGNAL(owner, COMSIG_ADD_MOOD_EVENT, "bipolar", /datum/mood_event/mood_swing_up)
+	else
+		to_chat(owner, "<span class='danger'>You feel the stress catching up to you. Sadness fills your heart!</span>")
+		SEND_SIGNAL(owner, COMSIG_ADD_MOOD_EVENT, "bipolar", /datum/mood_event/mood_swing_down)
+	addtimer(CALLBACK(src, .proc/change_mood_swing), 2 MINUTES)
+
+/datum/brain_trauma/psychological/bipolar/on_gain()
+	change_mood_swing()
+	ADD_TRAIT(owner, TRAIT_BIPOLAR, PSYCH_TRAIT)
+	..()
+
+/datum/brain_trauma/psychological/bipolar/on_lose()
+	REMOVE_TRAIT(owner, TRAIT_BIPOLAR, PSYCH_TRAIT)
+	..()
+
+/datum/brain_trauma/psychological/schizophrenia
+	name = "Schizophrenia"
+	desc = "Bread and butter of every psychopath"
+	gain_text = "<span class='danger'>You feel reality warping to your perception</span>"
+	lose_text = "<span class='notice'>World shifts back to its original place</span>"
+	scan_desc = "Electric imaging suggests a mental disorder is present"
+	cure_points = 1000
+	lithiated_mod = 5
+	control_mod = 3
+	fearless_mod = -3
+	relaxed_mod = -2
+
+/datum/brain_trauma/psychological/schizophrenia/on_life()
+	..()
+	if(prob(25))
+		owner.Jitter(4)
+		owner.derpspeech = min(owner.derpspeech + 5, 25)
+		owner.stuttering = min(owner.stuttering + 5, 25)
+
+/datum/brain_trauma/psychological/schizophrenia/on_lose()
+	owner.stuttering = 0
+	owner.derpspeech = 0
+	..()
+
+/datum/brain_trauma/psychological/schizophrenia/paranoid
+	name = "Paranoid schizophrenia"
+	desc = "Mental disorder that is characterized by auditory and visual hallucinations"
+	gain_text = "<span class='danger'>You hear whispers behind you.</span>"
+	lose_text = "<span class='notice'>Voices stop responding.</span>"
+	scan_desc = "Electric imaging suggests a mental disorder is present"
+	cure_points = 1300
+	lithiated_mod = 5
+	control_mod = 3
+	fearless_mod = -3
+	relaxed_mod = -2
+
+/datum/brain_trauma/psychological/schizophrenia/paranoid/on_life()
+	..()
+	owner.hallucination += 75 //Schizophernia is supposed to be fucking hard. This ensures it
+	owner.confused += 1
+	if(prob(0.5))
+		owner.Unconscious(80)
+
+
+
+/datum/brain_trauma/psychological/schizophrenia/paranoid/on_gain()
+	ADD_TRAIT(owner, TRAIT_PARANOID, PSYCH_TRAIT)
+	..()
+
+/datum/brain_trauma/psychological/schizophrenia/paranoid/on_lose()
+	REMOVE_TRAIT(owner, TRAIT_PARANOID, PSYCH_TRAIT)
+	..()
+
+#define MOANING_SPREE 0
+#define BLIND_SPREE 1
+#define UNCONCIOUS_SPREE 2
+#define SHOUTING_SPREE 3
+#define SPASM_SPREE 4
+#define MUTE_SPREE 5
+
+/datum/brain_trauma/psychological/schizophrenia/delusional
+	name = "Delusional schizophrenia"
+	desc = "Mental disorder that is characterized by auditory and visual hallucinations"
```

Comment:
change this to be different to the description of the previous brain trauma in this list

Reply:
alright, I'm stopping here for tonight

tomorrow, I'll go over the more serious issues with this PR instead of continuing to proofread it for you